tests/test_simulate1d.py

```diff
@@ -44,7 +44,11 @@ def test_rolled_gradient(simulated_field):
 def test_chi2():
     # A multivariate normal vector, x, with covariance matrix S, shall satisfy x'(S^-1)x ~ chi2(x.size)
     # This test makes an approximate check of this assumption. Parameters are taken from a Hufsa case.
-    from scipy.stats import chi2
+    try:
+        from scipy.stats import chi2
+    except ImportError:
+        from helpers import chi2
+
```

Comment:
Why not just use the `helpers` version always now? Is it worse?

Reply:
If it gives the same results, even if it's a bit slower, I think removing scipy from the dependency array  makes it worth it.

Reply:
My main intention for keeping the original `scipy` in there, is to have ha reality check on my implementation as well.
I imagine there might be more tests at some point, which may use `scipy`.

As an aside; some of the `utils` functions in `gaussianfft` does use `scipy`